tests/conftest.py

```
@@ -0,0 +1,129 @@
+    """
+    return make_can_addressing_information(example_can_addressing_format)
+
+
+@fixture
+def parametrized_can_addressing_information(request):
+    """Example value of CAN Addressing Information for CAN Addressing Format used."""
+    addressing_format = request.node.funcargs['addressing_format']
+    return make_can_addressing_information(addressing_format)
+
+
+@fixture()
```

Ruff (0.12.2)

Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)